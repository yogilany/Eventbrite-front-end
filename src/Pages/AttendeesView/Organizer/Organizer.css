.bg {
  background-image: url("../../../assets/Organizer//AnasOrgblur.jpg");
  height: 750px;
  width: 100%;
  display: flex;
  background-position: center;
  background-repeat: no-repeat;
  background-size: cover;
  position: relative;
}

.content {
  margin-top: 80px;
  width: 100%;
  position: absolute;
  margin: auto;
}

div.scrollmenu {
  background-color: transparent;
  overflow: auto;
  white-space: nowrap;
  margin-top: 30px;
  margin-left: 10%;
}

div.scrollmenu a {
  display: inline-block;
  font-weight: 600;
  color: #3d64ff;
  text-align: center;
  padding: 14px;
  text-decoration: none;
}

div.scrollmenu a:hover {
  text-decoration: underline;
}

.select {
  color: black;
  margin-top: 30px;
  margin-left: 10.5%;
  font-size: 40px;
  font-weight: 600;
}

.uporpast {
  margin-top: 20px;
  margin-left: 10.5%;
  font-size: 40px;
  font-weight: 600;
}

.Cards {
  margin-right: 8%;
  margin-left: 8%;
}
