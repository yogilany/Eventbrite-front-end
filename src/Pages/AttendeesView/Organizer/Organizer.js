import React from "react";
import ToggleButton from "react-bootstrap/ToggleButton";
import ToggleButtonGroup from "react-bootstrap/ToggleButtonGroup";
import "./Organizer.css";
import Head from "./Components/Head";
import Myphoto1 from "../../../assets/Organizer/AnasOrg.jpg";
import Events from "./Components/Events";
import { Container } from "react-bootstrap";

// make function

function Organizer() {
  return (
    <Container>
      <div className="bg">
        <div className="content">
          <Head
            photo={Myphoto1}
            name="Anas Org for mastering sorcery"
            followers="120"
            totalevents="2"
            info="This is Anas Organization for mastering sorcery around the world, our goal here is to develop newborn sorcerers with magical abilities, join us in our journey and we promise, you wont regret it :)"
          />
          <div className="lowersection">
            <div class="scrollmenu">
              <a href="#">Events</a>
            </div>
            <p className="select">Events</p>
            <div className="uporpast">
              <ToggleButtonGroup type="radio" name="options" defaultValue={1}>
                <ToggleButton id="tbg-radio-1" value={1}>
                  Upcoming (1)
                </ToggleButton>
                <ToggleButton id="tbg-radio-2" value={2}>
                  Past (1)
                </ToggleButton>
              </ToggleButtonGroup>
            </div>
            <div className="Cards">
              <Events
                title="Dr Stephen Strange"
                date="Sunday, March 12, 2023"
                loc="Earth 616"
                price="Starts from 12.5 EGP"
              />
            </div>
          </div>
        </div>
      </div>
    </Container>
  );
}

export default Organizer;
