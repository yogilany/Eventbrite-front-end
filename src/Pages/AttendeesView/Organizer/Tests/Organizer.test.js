import React from "react";
import { render, shallow } from "enzyme";
import { screen } from "@testing-library/react";
import userEvent from "@testing-library/user-event";
import Organizer from "../Organizer";

describe("Organizer", () => {
  it("should render the component correctly", () => {
    const { container } = render(<Organizer />);
    expect(container).toMatchSnapshot();
  });
});
