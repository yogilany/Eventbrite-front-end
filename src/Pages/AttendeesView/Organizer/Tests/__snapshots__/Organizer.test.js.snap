// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Organizer should render the component correctly 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="bg"
    >
      <div
        class="content"
      >
        <div
          class="profilecard"
        >
          <div
            class="avatar"
          >
            <div
              style="background: rgb(27, 61, 80); width: 180px; height: 180px; border-radius: 50%; display: flex; justify-content: center; align-items: center; transition: all 0.5s ease; box-shadow: inherit;"
            >
              <img
                style="width: 180px; height: 180px; border-radius: 50%;"
              />
            </div>
          </div>
          <div
            class="Orgname"
          >
            <p>
              undefined undefined
            </p>
          </div>
          <div
            class="Btns"
          >
            <button
              class="follow btn btn-outline-primary btn-lg"
              type="button"
            >
              Follow
            </button>
             
            <button
              class="contact btn btn-outline-secondary btn-lg"
              type="button"
            >
              Contact
            </button>
             
          </div>
          <div
            class="nums"
          >
            <div
              class="followers"
            >
              <span>
                <strong>
                  120
                </strong>
              </span>
              <p>
                Followers
              </p>
            </div>
            <div
              class="totalevents"
            >
              <span>
                <strong>
                  2
                </strong>
              </span>
              <p>
                Total Events
              </p>
            </div>
          </div>
          <div
            class="orginfo"
          >
            <p>
              This is Anas Organization for mastering sorcery around the world, our goal here is to develop newborn sorcerers with magical abilities, join us in our journey and we promise, you wont regret it :)
            </p>
          </div>
          <div
            class="socialmedia"
          >
            <link
              href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
              rel="stylesheet"
            />
            <a
              class="fa fa-facebook"
              href="#"
            />
            <a
              class="fa fa-twitter"
              href="#"
            />
          </div>
        </div>
        <div
          class="lowersection"
        >
          <div
            class="scrollmenu"
          >
            <a
              href="#"
            >
              Events
            </a>
          </div>
          <p
            class="select"
          >
            Events
          </p>
          <div
            class="uporpast"
          >
            <div
              class="btn-group"
              role="group"
            >
              <input
                autocomplete="off"
                checked=""
                class="btn-check"
                id="tbg-radio-1"
                name="options"
                type="radio"
                value="1"
              />
              <label
                class="btn btn-primary"
                data-testid="tbg-radio-1"
                for="tbg-radio-1"
                tabindex="0"
              >
                Upcoming (1)
              </label>
              <input
                autocomplete="off"
                class="btn-check"
                id="tbg-radio-2"
                name="options"
                type="radio"
                value="2"
              />
              <label
                class="btn btn-primary"
                data-testid="tbg-radio-2"
                for="tbg-radio-2"
                tabindex="0"
              >
                Past (1)
              </label>
            </div>
          </div>
          <div
            class="Cards"
          >
            <div
              class="wholecard"
            >
              <div
                class="bgimg"
              >
                <div
                  class="likebtn"
                >
                  <a
                    class="btn lovebtn"
                    href="#"
                    title="Add to Favourites"
                  >
                    <span>
                      ‚ù§
                    </span>
                  </a>
                </div>
                <div
                  class="content"
                >
                  <p
                    class="title"
                  >
                    Dr Stephen Strange
                  </p>
                  <p
                    class="date"
                  >
                    Sunday, March 12, 2023
                  </p>
                  <p
                    class="location"
                  >
                    Earth 616
                  </p>
                  <p
                    class="price"
                  >
                    Starts from 12.5 EGP
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
